src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -229,12 +229,17 @@ export class CalcitePickListItem {
     if (!icon) {
       return null;
     }
-
-    const iconName = icon === ICON_TYPES.circle ? ICONS.circle : ICONS.checked;
-
+    
     return (
-      <span class={CSS.icon}>
-        <calcite-icon scale="s" icon={iconName} />
+      <span
```

Comment:
@driskull Thoughts on this?
Makes single-select pick-list-item look more like some of the other Calcite things.
![Screen Shot 2020-08-18 at 2 19 29 PM](https://user-images.githubusercontent.com/12503298/90566798-f3665700-e15d-11ea-99bf-9396b43260d5.png)
And doesn't use the weird icon.

Reply:
Could selected ones have a slightly darker text / treatment (dropdown bumps up a font weight value)?

Reply:
Looks good to me 👍

Reply:
@macandcheese I'm down to work on that. Do you mind if we do more style alignments after this PR is in? There are a number of other areas that could use work to better align UI.

Reply:
Yeah no rush, just wanted to provide example of what we do with similar UI. Looking good!
<img width="263" alt="Screen Shot 2020-08-18 at 2 35 33 PM" src="https://user-images.githubusercontent.com/4733155/90568118-198cf680-e160-11ea-98bc-fa503e8e32b4.png">

Reply:
Noice.

Reply:
Looks like the dot sizes need some tweaking too.

Reply:
@macandcheese Updated the dot to use the same method that dropdown uses.